apps/pie-storybook/stories/pie-checkbox.stories.ts

```diff
@@ -165,7 +165,7 @@ const ExampleFormTemplate: TemplateFunction<CheckboxProps> = ({
     <form id="testForm">
         <pie-checkbox
             .value="${value}"
-            name="${ifDefined(name)}"
+            ?name="${ifDefined(name)}"
```

Comment:
Despite these being optional props, they weren't marked as such. Updated both stories to reflect this 👍🏼